DTS-STN/dts-rmp
# Home button with dashboard link added.

# Description

Added:
- Home button
- Dashboard link

## Test Instructions

1. Click the dashboard link. It should send you to the dashboard page.
2. Test with different browsers with all width to verify the layout.

## Help Requested


## Checklist
